package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Legal Aid",
   "main": "index.js",
   "scripts": {
-    "lint": "gulp eslint",
+    "lint": "node node_modules/gulp/bin/gulp eslint",
```

Comment:
What are you trying to achieve here? use `gulp lint`.
Define your task as lint. And remember to install gulp globally on your system.